test/unit_tests/braket/timings/test_time_series.py

```diff
@@ -54,6 +54,123 @@ def test_get_sorted(default_values, default_time_series):
     assert default_time_series.values() == [item[1] for item in sorted_values]
 
 
+def test_constant_like():
+    times = list(range(10))
+    constant_ts = TimeSeries.constant_like(times, constant=3.14)
+    assert times == constant_ts.times()
+    assert constant_ts.values() == [3.14] * len(times)
+
+
+def test_periodic_signal():
```

Comment:
Please test with different "boundary" types. I suggest you use "@pytest.mark.parametrize"